packages/react-compass/src/button/button.styles.ts

```diff
@@ -75,6 +75,7 @@ export const StyledButtonContent = styled('div', {
     display: 'flex',
     alignItems: 'center',
     justifyContent: 'center',
+    margin: 'auto',
```

Comment:
Should use justify content instead of margin auto. Remember to handle only the case button has icons.